### ISSUE TITLE
Strange Connections to Outside

### ISSUE THREAD

**Issue by RolfBremer** [hikerpig/pintora#265]
The following activity diagram has some strange lines to the outside:

![image](https://github.com/hikerpig/pintora/assets/46456966/fb6bb830-d431-4f49-8dee-afc60652a49d)

```
activityDiagram
    title: Test Activity
    start
        partition allmystuff {
            if(source newer that target?) then
                :Convert Source to Target;
            else (no)
            endif

            partition #0000ff repeat {
                if(source newer that target?) then
                    :Convert Source to Target;
                else (no)
                endif
            }
        }
    end
```

**Comment by hikerpig** [comment]
I see, it looks really weird when 
1. there is a condition branch without statements
2. the condition is inside a partition

I'll find some time to fix it

**Comment by hikerpig** [comment]
This is fixed is v0.7.3
